pom.xml

```diff
@@ -89,6 +89,21 @@
             </dependency>
         </dependencies>
     </dependencyManagement>
+    
+    <dependencies>
```

Comment:
I'm inclined to blame your Eclipse. I guess we can include them for now but would be good if you worked out the proper configuration/why Eclipse is complaining.